examples/ansible/setup_libvirt_host.yml

```diff
@@ -25,36 +29,35 @@
         name: libvirtd
         wait_for:
           path: /var/run/libvirt/libvirt-sock
-
   tasks:
-
     - name: Retrieve image for workload {{ workload.name }}
       become: true
       containers.podman.podman_image:
         name: "{{ workload.image }}"
         state: present
       register: retrieve_workload_image
+      failed_when: retrieve_workload_image is failed
 
     - name: Install tools for workload {{ workload.name }}
       become: true
       ansible.builtin.command: >-
         podman container runlabel {{ workload.labels.install }} {{ workload.image }}
       register: workload_runlabel_install
-      changed_when:
-        - ('already exist' not in workload_runlabel_install.stdout)
+      changed_when: "'already exist' not in workload_runlabel_install.stdout"
+      failed_when: workload_runlabel_install is failed
 
     - name: Install required packages, if any, for workload {{ workload.name }}
       become: true
       ansible.builtin.package:
         name: "{{ workload.required_pkgs }}"
         state: present
       register: required_pkgs_install
-      when:
-        - ((workload.required_pkgs | default([])) | length) > 0
+      when: ((workload.required_pkgs | default([])) | length) > 0
```

Comment:
This is an unnecessary formatting/whitespace change to the existing; changes to existing code should be minimised when proposing new changes, and new code should be written to conform to the same coding style/pattern as the original code.

If there is a desire to change the coding style of the code that should be handled as a separate PR that makes no functional changes to the code.